source/renderer/app/containers/etc/LoadingPage.js

```diff
@@ -32,7 +31,7 @@ export default class LoadingPage extends Component<InjectedProps> {
           allowedTimeDifference={ALLOWED_TIME_DIFFERENCE}
           isConnecting={isConnecting}
           syncPercentage={syncPercentage}
-          isLoadingDataForNextScreen={isLoadingWallets}
+          isLoadingDataForNextScreen={!isSyncing || isSynced}
```

Comment:
@DominikGuzei I believe we should take this opportunity to refactor `isSyncing` and `isSynced` logic in the `NetworkStatusStore`: https://github.com/input-output-hk/daedalus/blob/development-webpack-fork/source/renderer/app/stores/NetworkStatusStore.js#L140-L151